addons/google_calendar/models/calendar_attendee.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+from openerp import api, fields, models
+
+
+class CalendarAttendee(models.Model):
+    _inherit = 'calendar.attendee'
+
+    google_internal_event_id = fields.Char('Google Calendar Event Id')
+    oe_synchro_date = fields.Datetime('Odoo Synchro Date')
```

Comment:
Doesn't make any difference, first arg is string :), for code consistency we should write string +1